docs/product-installation-guide/src/main/asciidoc/install-testing-proc.adoc

```diff
@@ -4,34 +4,54 @@
 After you have installed {PRODUCT}, create an asset to verify that the installation is working.
 
 .Procedure
-. Enter  the following command to start {KIE_SERVER}:
+. In a terminal window, enter the following command to start {KIE_SERVER}:
 +
 [source]
 ----
-EAP_HOME/bin/standalone.sh
+_EAP_HOME_/bin/standalone.sh
 ----
 
-. In a web browser, enter `localhost:8080/decision-central`. If {PRODUCT} has been configured to run from a domain name, substitute `localhost` for the domain name, for example:
-+
-http://www.example.com:8080/decision-central
+. In a web browser, enter 
+ifdef::PAM[]
+`localhost:8080/business-central`.
+endif::[]  
+ifdef::DM[]
+`localhost:8080/decision-central`.
+endif::[]  
+
+* If {PRODUCT} has been configured to run from a domain name, substitute `localhost` for the domain name, for example:
 +
-If {PRODUCT} has been configured to run in a cluster, substitute `localhost` for the IP address of a particular node, for example:
+ifdef::PAM[]
+`http://www.example.com:8080/business-central`
+endif::[]  
+ifdef::DM[]
+`http://www.example.com:8080/decision-central`
+endif::[] 
+* If {PRODUCT} has been configured to run in a cluster, substitute `localhost` for the IP address of a particular node, for example:
 +
+ifdef::PAM[]
+`http://<node_IP_address>:8080/business-central`
+endif::[]  
+ifdef::DM[]
 `http://<node_IP_address>:8080/decision-central`
+endif::[] 
 . Enter the admin user credentials that you created during installation. The {CENTRAL} home page appears.
 . Select *Menu* -> *Design* -> *Projects*.
 . Click *Try Samples*.
-. Click *mortages*. The Assets window appears.
-. Click *Create New Asset*.
+. Click *Mortages* and click *OK*. The Assets window appears.
+. Click *Add Asset*.
 . Select *Data Object*.
-. Enter MyDataObject in the *Name* field and click *OK*.
-. Click *Spaces* -> *myteam* -> *mortgages* and confirm that MyDataObject is in the list of assets.
-. If you are verifying a clustered installation, enter the following URL, where `<node_IP_address>` is the address of different node. Enter same credentials that you used with the {CENTRAL} where you created the MyDataObject asset.
+. Enter MyDataObject in the *Data Object* field and click *OK*.
```

Comment:
This is entirely up to you since it's consistent throughout and I think that matters more, but I would put `` around MyDataObject as it's a configurable attribute. I don't know if there's a widely accepted convention on this though as our Asciidoc Mark-Up Quick Reference guide hasn't been updated since last year and the only other one I know of is the EAP one.